common/core/web/input-processor/src/text/prediction/languageProcessor.ts

```diff
@@ -255,7 +255,7 @@ namespace com.keyman.text.prediction {
             // the input will be automatically rewound to the preInput state.
             transform: original.transform,
             // The ID part is critical; the reversion can't be applied without it.
-            transformId: original.token, // reversions use the additive inverse.
+            transformId: -original.token, // reversions use the additive inverse.
```

Comment:
I'm trying to understand the significance of this change. Is it a bug fix -- the comment suggests as such? Or is it just to support the other changes you are making now?

Reply:
Bug fix.  I think at some point, the intent was to use the base ID (which already used the additive inverse), but `transformID` is the field in active use for ID checks now.  Not sure when reversions broke.  Anyway, that comment was very important in figuring out why things were broken.